src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs

```diff
@@ -368,10 +368,13 @@ public static DateValue NumberToDate(IRContext irContext, NumberValue[] args)
             return new DateValue(irContext, date);
         }
 
-        public static DateTimeValue NumberToDateTime(IRContext irContext, NumberValue[] args)
+        public static FormulaValue NumberToDateTime(EvalVisitor runner, EvalVisitorContext context, IRContext irContext, NumberValue[] args)
```

Comment:
>FormulaValue [](http://example.com/codeflow?start=22&length=12)

Why change the return type? We always return a DateTimeValue from this method.

Reply:
Due to previous iteration changes which I've finally reverted, I'll fix that